### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by hjheath**

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://110-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://110-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This project adheres to the Node default version scheme, meaning It's safe to us
 The standard for the caret can [be found here](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).
 Breaking changes result in a different major. UI changes that might break customizations on top of the sdk, will be treated as breaking changes too.
 
-## [next version]
+## [0.11.0]
```
Comment (rfreitas) [inline CHANGELOG.md]: missing the diff link to the previous version

**Comment by rfreitas** [comment]
Travis automatic deployment: https://110-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://110-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md (hunk 1 above)
Comment (hjheath) [inline CHANGELOG.md]: fixed

**Pull request merged**